SocialFilter/sections/css_extended.txt

```diff
@@ -17,6 +17,7 @@ quantamagazine.org#?#.post__sidebar__content > div.post__sidebar__content__inner
 mechacomic.jp#?##main > section.c-content > div.area_btn_sns:upward(1)
 corp.helpfeel.com#?#body > .rebranding__section--last > .w-container:has(> .rebranding__share__buttons)
 newsyou.info#?#nofollow > a[href^="https://t.me/joinchat"]:upward(1)
+hinode.pics#?##right_wrap > div.right_plugin_wrap:contains(ソーシャルボタン)
```

Comment:
You can create a rule without using `:contains()`.
```
hinode.pics#?##right_wrap > div.right_plugin_wrap:has(> div.right_plugin_body > div#right_plugin_social_buttons_wrap)
```

Reply:
Thank you